apps/opencs/model/world/collection.hpp

```diff
@@ -120,6 +120,8 @@ namespace CSMWorld
 
             virtual const Record<ESXRecordT>& getRecord (int index) const;
 
+            virtual Record<ESXRecordT>& getRecord (int index);
```

Comment:
I needed a way to get a reference to a record to modify (i.e. non-const version), passed as a parameter, but couldn't think of a way to get it working with setRecord.  I'll have another think about it. 

It's being used here:

    template<typename ESXRecordT, typename IdAccessorT>
    void SubCellCollection<ESXRecordT, IdAccessorT>::addNestedRow(int row, int column, int position)
    {
        getAdapter(Collection<ESXRecordT, IdAccessorT>::getColumn(column))->addNestedRow(
                Collection<ESXRecordT, IdAccessorT>::getRecord(row), position);
    }